pull_request_template.md

```diff
@@ -33,3 +33,5 @@
 
 - [ ] I have self-reviewed my code and added comments where relevant.
 - [ ] I have updated any relevant documentation.
+- [ ] I have incremented the version number in the manifest.json file.
```

Comment:
I sometimes forget to do this step so wanted to update this pull request template for my browser extension to remind me to do this.